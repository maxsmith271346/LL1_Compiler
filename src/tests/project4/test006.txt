main
int x,y,z;
{
	let a = 2;
}.
