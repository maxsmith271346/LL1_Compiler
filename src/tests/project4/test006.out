Error parsing file.
ResolveSymbolError(4,6)[Could not find a.]

